test/unit/org/apache/cassandra/service/ActiveRepairServiceTest.java

```diff
@@ -387,10 +407,13 @@ public void testRejectWhenPoolFullStrategy() throws InterruptedException
             {
                 // expected
             }
+
             // allow executing tests to complete
             blocked.signalAll();
-            completed.await(10, TimeUnit.SECONDS);
+            completed.await(11, TimeUnit.SECONDS);
```

Comment:
nit: could we get a comment here please? or even better make `10` a well-named constant so we understand that's the wait happening inside the task.

Reply:
+1